libvast/vast/type.hpp

```diff
@@ -869,6 +869,10 @@ struct sum_type_access<vast::type> {
 
 namespace vast {
 
+/// Identifies default-constructed `type` objects.
+static constexpr uint8_t invalid_type_tag = 255;
```

Comment:
There is already an `invalid_type_id` at line 76. Why can't we use that instead?